src/test/java/com/napier/sem/DAO_IntegrationTests.java

```diff
@@ -253,6 +275,23 @@ void topNCitiesIn_allAreasMatchFilter ()
         for (Record city : cities) assertEquals("United Kingdom", city.getCountry());
     }
 
+    // Tests that the results are listed in descending order.
+    @Test
+    void topNCitiesIn_topNCitiesPopulationIsInDescendingOrder()
+    {
+        // Given
+        int n = 5;
+
```

Comment:
In some Descending Order tests the given has been the area name and in some it is n, should this not be consistent? It could be both n and the area name for methods like this

Reply:
the given is actually not very clear, because what we're testing is just the order they are showing up in. i'd maybe be in favour of removing the given for all of these